### PR TITLE
docs(blog): add news blog for third community days

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR adds the initial news blog for the community days. The content will be changed in the future after the agenda is fixed.

Until now the page would look like this

image

image

image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by tom-rm-meyer-ISST — DISMISSED** [review]
LGTM!

File: blog/2024-12-05-community-days.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Join Us for the Exciting Third Eclipse Tractus-X Community Days!
```
Comment (evegufy) [inline blog/2024-12-05-community-days.mdx:2]:
```suggestion
title: Join Us for the Third Eclipse Tractus-X Community Days!
```

just as a comment, I'd keep it precise and it's of course exciting, but IMO no need to say that in the title.

**Review by saudkhan116 — APPROVED** [review]
Looks fine to me!

**Comment by maximilianong** [comment]
Nice! Thanks Stephan for creating / preparing this PR.

**Review by evegufy — APPROVED** [review]

**Review by Phil91 — APPROVED** [review]
lgtm

**Comment by matbmoser** [comment]
LGTM

**Pull request merged**